README.md

```diff
@@ -67,7 +67,6 @@ simple 3D example.
 git clone --recursive https://github.com/blackwer/baobzi.git
 cd baobzi
 mkdir build
-cd build
 # Don't supply -march!!! Builds with CPU dispatch
 cmake -DBAOBZI_BUILD_MATLAB=True -DCMAKE_BUILD_TYPE=RelWithDebInfo -DCMAKE_INSTALL_PREFIX=$HOME/local
```

Comment:
Given the build comment. The cmake command should have a trailing `..` so my bad :)